config/arkscan.php

```diff
@@ -56,7 +58,16 @@
             'base58Prefix'        => intval(env('ARKSCAN_NETWORK_BASE58_PREFIX', 30)),
 
             'contract_addresses' => [
-                'consensus' => env('ARKSCAN_CONSENSUS_CONTRACT_ADDRESS', '0x535B3D7A252fa034Ed71F0C53ec0C6F784cB64E1'),
+                'consensus' => env('ARKSCAN_CONTRACT_CONSENSUS_ADDRESS', '0x535B3D7A252fa034Ed71F0C53ec0C6F784cB64E1'),
+            ],
+
+            'contract_methods' => [
+                'transfer'               => env('ARKSCAN_CONTRACT_TRANSFER_METHOD', 'a9059cbb'),
+                'batch_transfer'         => env('ARKSCAN_CONTRACT_BATCH_TRANSFER_METHOD', '88d695b2'),
```

Comment:
a new contract was created for it, and named after multipayments again: https://github.com/ArkEcosystem/mainsail/blob/6957ff682e789380fbbb6c7db7d39f62680ca719/contracts/src/multi-payment/MultiPayment.sol

we'll need to have a look at the signature for that and point it to the updated contract

Reply:
I've adjusted this but I'm waiting for confirmation that the method id/mapping is correct

Reply:
Joshua confirmed what I've changed it to is correct